site/content/integrate/slash-commands/custom/_index.md

```diff
@@ -0,0 +1,137 @@
+| `text`               | [Markdown-formatted](https://docs.mattermost.com/messaging/formatting-text.html) message to display in the post.                                                                                                                                                                                                                                     | If `attachments` is not set, yes |
+| `response_type`      | Set to blank or `ephemeral` to reply with a message that only the user can see. <br/> Set to `in_channel` to create a regular message.<br/> Defaults to `ephemeral`.                                                                                                                                                                                 | No                               |
+| `username`           | Overrides the username the message posts as.<br/> Defaults to the username set during webhook creation or the webhook creator's username if the former was not set.<br/> Must be enabled [in the configuration](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-usernames).                        | No                               |
+| `channel_id`         | Overrides the channel to which the message gets posted.<br/> Defaults to the channel in which the command was issued.                                                                                                                                                                                                                                | No                               |
+| `icon_url `          | Overrides the profile picture the message posts with.<br/> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br/> Must be enabled [in the configuration](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons). | No                               |
+| `goto_location`      | A URL to redirect the user to. Supports many protocols, including `http://`, `https://`, `ftp://`, `ssh://` and `mailto://`.                                                                                                                                                                                                                         | No                               |
+| `attachments`        | [Message attachments]({{< ref "/integrate/reference/message-attachments" >}}) used for richer formatting options.                                                                                                                                                                                                                                    | If `text` is not set, yes        |
```

Comment:
Since this is one of the only other possible required arguments, I think it should get moved up under `text`.